src/style.css

```diff
@@ -1 +1,95 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+
+}
+
+body {
+  margin: 0;
+}
+
+.site-header {
+  max-width: 1200px;
+}
+
+.inner-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-decoration: none;
+  flex-wrap: wrap;
+  padding: 0 50px;
+  height: 60px;
+  box-sizing: border-box;
+}
+
+.logo_wrapper {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav {
+  box-sizing: border-box;
+}
+
+.nav__item {
+  display: inline-block;
+  height: 60px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: block;
+  position: relative;
+  height: 100%;
+  line-height: 60px;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: black;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.nav_link:hover {
+  color: hsl(193, 100%, 43%);
+}
+
+.nav_link:active {
+  color: hsl(193, 100%, 43%);
+}
```

Comment:
We can combine selector with the same styles using comma

Reply:
Thank you! Fixed suggestion